Modules/Core/SpatialObjects/test/itkImageMaskSpatialObjectGTest.cxx

```diff
@@ -153,13 +153,13 @@ TEST(ImageMaskSpatialObject, AxisAlignedBoundingBoxIsEmptyWhenAllPixelsAreZero)
 {
   // Test 2D images:
   Expect_AxisAlignedBoundingBoxRegion_is_empty_when_all_pixel_values_are_zero<double>(
-    itk::ImageRegion<2>{ itk::Index<2>{}, itk::Size<2>::Filled(1) });
+    itk::ImageRegion<2>{ itk::Index<2>{{0, 0}}, itk::Size<2>::Filled(1) });
```

Comment:
It's a silly warning, because the original code is really the way it is meant to be, in modern C++. However, in order to please this (obviously outdated) compiler, I would prefer using parentheses, instead of curly braces, which is old C++03 syntax: `itk::Index<2>()`.

Reply:
Ah, great suggestion -- done!